code/game/objects/items/stacks/medical.dm

```diff
@@ -58,8 +58,8 @@
 		user.visible_message("<span class='green'>[user] applies \the [src] on [C]'s [affecting.name].</span>", "<span class='green'>You apply \the [src] on [C]'s [affecting.name].</span>")
 		var/brute2heal = brute
 		var/burn2heal = burn
-		if(user?.mind?.get_skill_speed_modifier(/datum/skill/medical))
-			var/skillmods = user.mind.get_skill_speed_modifier(/datum/skill/medical)
+		if(user?.mind?.get_skill_modifier(/datum/skill/medical, SKILL_SPEED_MODIFIER))
+			var/skillmods = user.mind.get_skill_modifier(/datum/skill/medical, SKILL_SPEED_MODIFIER)

```

Comment:
seems like you could avoid calling this twice

Reply:
good point

Reply:
top tier web edit

also good to see you again

Reply:
thank you! 👍